### PR TITLE
EPMEDU-929: Fix logout confirmation toast is not displayed

### PR DESCRIPTION
**Pull request by igorTimofiejczyk**

### CODE REVIEW

File: animeal/src/App/AppCoordinator.swift [hunk 1]
```diff
@@ -99,6 +106,17 @@ final class AppCoordinator: AppCoordinatable {
         }
     }
 
+    private func handleBackwardAction(_ action: HomeFlowBackwardAction) {
+        switch action {
+        case .shouldShowToast(let title):
+            DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
```
Comment (marek-stoma) [inline animeal/src/App/AppCoordinator.swift]: do we need asyncAfter here?

File: animeal/src/Flows/Main/Modules/HomeModule/Home/View/HomeViewController.swift [hunk 2]
```diff
@@ -173,14 +173,19 @@ private extension HomeViewController {
         var closestLocation: CLLocation?
         var smallestDistance: CLLocationDistance?
 
+        let minPermittedDistinationInMeters: CLLocationDistance = 10000
```
Comment (marek-stoma) [inline animeal/src/Flows/Main/Modules/HomeModule/Home/View/HomeViewController.swift]: Is this change related to the logout toast?

Comment (igorTimofiejczyk) [reply animeal/src/Flows/Main/Modules/HomeModule/Home/View/HomeViewController.swift]: Removed, will be added in separate PR

File: animeal/src/App/AppCoordinator.swift (hunk 1 above)
Comment (igorTimofiejczyk) [reply animeal/src/App/AppCoordinator.swift]: Decided to go easy way, but it didn't work :), however asyncAfter remover and added more robust logic instead.

**Review by marek-stoma — APPROVED** [review]
